src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IIsTypeExpression.cs

```diff
@@ -0,0 +1,163 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
+    {
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void TestIsOperator()
+        {
+            string source = @"
```

Comment:
Could you break up these tests? Large test blocks bring in a lot of unrelated IOperations and it is both hard to review, and can introduce a lot of meaningless noise when unrelated nodes change shape.

Reply:
> Could you break up these tests?

I support this request as well. Please use more targeted tests.

Reply:
Test has been ported from http://source.roslyn.io/#Roslyn.Compilers.CSharp.Emit.UnitTests/CodeGen/CodeGenOperators.cs,63ad66a1ca6ab689. Ideally, I would have just enhanced the existing test to have the operation tree verification, but the test lies in compiler emit tests project, and I think it is best to keep the operation tests in semantic tests project. Is it fine to clone and break down these tests into individual operation tests?

Reply:
I would expect to see tests that only verify bits of IOperation tree that are relevant to the change. This can be achieved in many different ways. For example, requesting and verifying IOperation for several specific syntax nodes in context of the same test is acceptable. What is not acceptable in my opinion, is to dump tree for big chunks of code that include all kind of constructs unrelated to the change. The main purpose of the added tests should be to verify effect of the change, the used approach makes it nearly impossible as the baseline is extremely hard to comprehend.

Reply:
@AlekseyTs I completely understand your request and the desire to have targeted tests. However, such tests are extremely hard to update for baseline changes, which are happening extremely frequently in our current development cycle. @333fred has a written a tool that bulk edits the baselines for tests written in a standard format - single operation tree verification in a unit test. If we start to hand craft tests where it is verifying multiple small pieces of operation tree, then we will need to spend lot of manual effort just updating the baselines for these tests, which is currently happening for almost every other merge conflict resolution. I would like to propose the following approach:

1. Search for existing compiler binding tests in the feature area
   1. If the tests exist and are small and targeted tests, extend them to verify the operation tree.
   2. If the tests exist but are large tests (like the TypeOf tests in this PR), we clone and break them into separate targeted IOperation tests.
   3. If the tests don't exist or are scattered all over the test projects, we create new IOperation unit tests.

So for the current PR, I would like to take approach 2.